metsrw/fsentry.py

```diff
@@ -298,7 +300,43 @@ def add_rightsmd(self, md, mdtype, mode="mdwrap", **kwargs):
         return self._add_metadata_element(md, "rightsMD", mdtype, mode, **kwargs)
 
     def add_dmdsec(self, md, mdtype, mode="mdwrap", **kwargs):
-        return self._add_metadata_element(md, "dmdSec", mdtype, mode, **kwargs)
+        """Add dmdsec.
+
+        Extension of _add_metadata_element that adds a dmdSec and updates the
+        previous dmdSecs of the same mdtype_othermdtype, marking them as
```

Comment:
I find the phrase `of the same mdtype_othermdtype` a bit confusing.

What do you think of something along the lines of `with the same MDTYPE and OTHERMDTYPE attribute values` instead?